src/main/java/frc/robot/subsystems/Indexer2022.java

```diff
@@ -0,0 +1,66 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotConstants;
+import frc.robot.logging.RobotLogManager;
+import frc.robot.motors.MotorControllerEncoder;
+import frc.robot.motors.MotorControllerFactory;
+import frc.robot.motors.MotorType;
+import org.apache.logging.log4j.Logger;
+
+/** The indexer subsystem, contains the indexer wheel and its motor only. */
+public class Indexer2022 extends SubsystemBase {
+  private static final Logger LOGGER = RobotLogManager.getMainLogger(Indexer2022.class.getName());
+
+  private final MotorControllerEncoder indexerMotor;
+
+  /** The indexer subsystem, contains the indexer wheel and its motor only. */
+  public Indexer2022() {
+    super();
+    indexerMotor =
+        MotorControllerFactory.create(
+            RobotConstants.get().indexer2022MotorID(), MotorType.TALON_SRX);
+
+    indexerMotor.setInverted(RobotConstants.get().indexer2022MotorInverted());
+  }
+
+  /**
+   * Idles the indexer wheel.
+   *
+   * <p>Keeps the wheel at a slow speed so that it can grab a ball once it reaches one.
+   */
```

Comment:
Pick a style for the comments. All of the other comments in this file are brief one-liners. Both styles are valid, but at least be consistent within the file. FWIW, the short version is probably sufficient.